docs/docs/guides/integrate/identity-providers/migrate.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Migrate from Generic Provider to specific Identity Provider
+sidebar_label: Migrate IDP
+---
+
+## Migrate Generic OIDC Provider
+
+You can migrate from a generic OIDC provider to the following supported templates:
+- AzureAD
+- Google
+
+To migrate you only have to use the [Migrate Generic OIDC Identity Provider (Instance)](/docs/apis/resources/admin/admin-service-migrate-generic-oidc-provider#migrate-generic-oidc-identity-provider) or [Migrate Generic OIDC Identity Provider (Organization)](/docs/apis/resources/mgmt/management-service-migrate-generic-oidc-provider#migrate-generic-oidc-identity-provider) call,
+which only changes the type of the provider and doesn't delete the already linked user to the provider.
+
+### Google Configuration
+
+The available configuration is described in [Google Configuration](./google).
+
+### AzureAD Configuration
+
+The available configuration is described in [AzureAD Configuration](./azure-ad).
+
+## Migrate with Terraform
+
+To migrate to a specific provider there are only a couple steps:
+- Have an already existing IDP, which has to be migrated as to not lose the existing links of users to users of the external IDP
+- Create a desired IDP as Terraform resource for example [Google](https://registry.terraform.io/providers/zitadel/zitadel/latest/docs/resources/idp_google).
+- Make the corresponding API call to [migrate the IDP](./migrate#google-configuration), save the ID of the IDP for the import
+- Before applying the Terraform resources again, import the new IDP resource.
+```bash
+#resource "zitadel_idp_google" "google" {
+#  name                = "Google"
+#  client_id           = "182902..."
+#  client_secret       = "GOCSPX-*****"
+#  scopes              = ["openid", "profile", "email"]
+#  is_linking_allowed  = false
+#  is_creation_allowed = true
+#  is_auto_creation    = false
+#  is_auto_update      = true
+#}
+
+# terraform import zitadel_idp_google.*resource_name* *id*:*client_secret*
+terraform import zitadel_idp_google.google 222302827723096428:GOCSPX-*****
+```
```

Comment:
It is actually only this 3 steps you have to perform, and the other two in the list are just notes or mentions

```suggestion
1. Create a desired IDP as Terraform resource for example [Google](https://registry.terraform.io/providers/zitadel/zitadel/latest/docs/resources/idp_google).
2. Make the corresponding API call to [migrate the IDP](./migrate#google-configuration), save the ID of the IDP for the import
3. Before applying the Terraform resources again, import the new IDP resource.
```bash
#resource "zitadel_idp_google" "google" {
#  name                = "Google"
#  client_id           = "182902..."
#  client_secret       = "GOCSPX-*****"
#  scopes              = ["openid", "profile", "email"]
#  is_linking_allowed  = false
#  is_creation_allowed = true
#  is_auto_creation    = false
#  is_auto_update      = true
#}

# terraform import zitadel_idp_google.*resource_name* *id*:*client_secret*
terraform import zitadel_idp_google.google 222302827723096428:GOCSPX-*****
```
```